app/src/main/java/com/asma/tasky/core/domain/util/Util.kt

```diff
@@ -0,0 +1,15 @@
+package com.asma.tasky.core.domain.util
+
+import java.time.LocalDateTime
+
+object Util {
+
+    fun getInitials(name: String): String {
+        val words = name.split(" ")
+        var initials = ""
+        words.forEach { word ->
+            initials += word.first().toString().uppercase()
+        }
+        return initials
+    }
```

Comment:
This implementation seems to not be correct. If the user has a middle name, you'll end up with 3 initials here. Also it'll likely break if there are 2 space chars in the name, since you'll end up with an empty word where you assume `first()` exists.
This would be a great and simple function to practice writing some tests for 👍